src/sentry/migrations/0171_backfill_cool.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 1.11.29 on 2021-03-11 06:29
+
+from django.db import migrations
+
+
+def backfill_is_cool(apps, schema_editor):
+    User = apps.get_model("sentry", "User")
+
+    for user in User.objects.all():
+        is_cool = user.id % 1337 == 0
+        user.update(is_cool=is_cool)
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
```

Comment:
Shouldn't run this automatically since we're potentially updating so many rows
```suggestion
    is_dangerous = True
```

Reply:
@wmak isn't the user table small enough where we shouldn't worry about `is_dangerous` when we are doing batched updates with `RangeQuerySetWrapperWithProgressBar`?

Reply:
Hmm I think it depends on how many rows of the user table we're changing, if we only update the ones that aren't already set correctly could be a small number of rows to update.